package.json

```diff
@@ -103,7 +103,7 @@
     "shelljs": "0.8.5",
     "simple-git": "3.17.0",
     "simple-plist": "1.4.0",
-    "source-map": "0.7.4",
+    "source-map": "0.8.0-beta.0",
```

Comment:
https://github.com/NativeScript/nativescript-cli/pull/5699#issuecomment-1327932750

I don't think we want that beta version.

Reply:
@rigor789 yes we already had that discussion a long time ago. It slipped through the PR.
I started using this because default (Which is dead old) was failing but i dont remember in which case